src/purchase-client.ts

```diff
@@ -0,0 +1,203 @@
+import { MuxySDK } from ".";
+import { CurrentEnvironment } from './util';
+import Config from './config';
+
+import { TwitchBitsProduct, TwitchBitsTransaction } from './twitch';
+
+// Muxy dev variable
+declare global {
+  interface Window {
+    MEDKIT_PURCHASABLE_ITEMS: Product[] | PromiseLike<Product[]>;
```

Comment:
I don't think this should ever by a Promise. What's the reasoning here?

Reply:
Doesn't have to.  Was making it work the same way as the twitch version.